src/ralph/defaults.py

```diff
@@ -37,3 +37,4 @@ class StorageBackends(Enum):
 ENVVAR_PREFIX = "RALPH"
 HISTORY_FILE = Path(environ.get("RALPH_HISTORY_FILE", APP_DIR / "history.json"))
 FS_STORAGE_DEFAULT_PATH = Path("./archives")
+SENTRY_DSN = None
```

Comment:
You should allow to override it using the environment.

Reply:
Thanks, I forgot to add that